sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/exemplar/NoExemplarReservoir.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.exemplar;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.metrics.data.Exemplar;
+import java.util.Collections;
+import java.util.List;
+
+/** Implementation of a reservoir that keeps no exemplars. */
+class NoExemplarReservoir implements ExemplarReservoir {
+
+  static final ExemplarReservoir NO_SAMPLES = new NoExemplarReservoir();
```

Comment:
how about just calling this "INSTANCE". This name clash threw me when I saw it being used. we could also hide it behind a static method as we've done in other places in the codebase.